docs/testing.rst

```diff
@@ -0,0 +1,58 @@
+Testing
+=======
+
+All tests can be run using `tox <https://tox.wiki/en/latest/>`_ simply by running the `tox` command.
+
+By default, tests are run against a local sqlite2 instance, however you can also use docker to test against MySQL and Postgres databases as follows.
+
+First, set environment variables.  Note that the ``IMPORT_EXPORT_TEST_TYPE`` must be set according to the type of tests you wish to run.  Set to 'postgres' for postgres tests, and 'mysql-innodb' for mysql tests.  If this environment variable is blank (or is any other value) then the default sqlite2 db will be used.
+
+.. code-block:: bash
+
+  export DJANGO_SETTINGS_MODULE=settings
+
+  export IMPORT_EXPORT_POSTGRESQL_USER=pguser
+  export IMPORT_EXPORT_POSTGRESQL_PASSWORD=pguserpass
+
+  export IMPORT_EXPORT_MYSQL_USER=mysqluser
+  export IMPORT_EXPORT_MYSQL_PASSWORD=mysqluserpass
+
+  # for testing against mysql db
+  export IMPORT_EXPORT_TEST_TYPE=mysql-innodb
+
+  # for testing against postgres db
+  export IMPORT_EXPORT_TEST_TYPE=postgres
```

Comment:
given that the `docker-compose up` command will start both postgresql and mysql, can we have a `IMPORT_EXPORT_TEST_TYPE=all`?

Also, what about having all these commands in `runtests.sh`, just explain databases run on docker but tox runs locally, this way we can have a simple way of running *all* tests (in addition to CI). This would allow for running tests for both postgresql and mysql if the above sentence doesn't make sense :)

Reply:
Good idea - will look into it.

Reply:
It is not possible to use 'all' because the database type is switched at runtime in `settings.py`, however it is possible to script this process, and I have made these changes in `runtests.sh`